src/api/operator/types.ts

```diff
@@ -84,24 +84,37 @@ export type RawTrade = {
   txHash: string
   orderUid: string
   buyAmount: string
+  executedBuyAmount: string
   sellAmount: string
+  executedSellAmount: string
+  executedFeeAmount: string
   sellAmountBeforeFees: string
   buyToken: string
   sellToken: string
+  executionTime: string
+  kind: OrderKind
+  surplusAmount: string
+  surplusPercentage: string
```

Comment:
Where is this data coming from?
They are not part of the api response.
Please make sure this type contains only data from the API.

Reply:
@alfetopito I've confused the fields from the orders api response.
I will remove those fields. But then I have some questions.
- Since the trades API does not return the fields like executedBuyAmount, executedSellAmount, executedFeeAmount, how we can calculate the executionPrice value?
- Also surplus does not exist in the api response. Then where should we get the value from?
- The same question for executionTime.

Reply:
Yes, that's something I know it's a problem.
For now, the plan is to get those values from individual `/order/<id>` calls for every single trade.
Not optimal, but that's the current db design.

For this reason, @henrypalacios 's idea to have a context shared between orders and trades table is super useful.
You can look up the given orderId on the context first and only if it doesn't exist query the API endpoint.

Anyway, I'll have a meeting with backend guys tomorrow to see if we can improve the data returned on these queries, but for the trades endpoint I don't expect much will change.

Reply:
@alfetopito then what do you think if we integrate the trades table with the actual API in this PR and close the other PR?
I guess it would be unnecessary to mock the order/trade data and fetch orders from the mock.
I could be not helpful to simplify the PR rather than integrating with real API.

Reply:
I still think they should be separated.
This PR is meant to have the UI only, using storybook, independent of the data source.

You can use static data as input on the stories. There's no need for a mock as far as I can tell. 

You define what's the interface the UI component will have, and  - even if the data from the API is not in the given format - you can transform it before passing it to the component.

Keep the UI data representation isolated from the API data representation. So both can be updated independently.

Also, this PR is not too far from being done.
I think once you have all the fields required it should be good to merge.